test/Driver/static-stdlib-autolink-linux.swift

```diff
@@ -0,0 +1,27 @@
+// Statically link a program with concurrency module
+// REQUIRES: OS=linux-gnu
```

Comment:
This is very narrow. I suppose it's because of the need for ldd?

Reply:
Yes, I added this requirement for ldd checks. I've removed this requirement and run ldd tests on only linux.